### PR TITLE
Fixing scale max_snap issue

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Fixing scale max_snap issue

Problem:
Filling cluster to 100 % and making it unusable

Solution:
We have controlled to use only 50 % in the available space for pool and running the test

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-RY26IR/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_scale/max_smallfile_snap.py [hunk 1]
```diff
@@ -6,18 +6,16 @@
...
 
-def get_available_space(client, fs_name="cephfs"):
-    out, rc = client.exec_command(
-        sudo=True, cmd=f"ceph fs status {fs_name} --format json"
-    )
-    output = json.loads(out)
-    return next(
-        (pool["avail"] for pool in output["pools"] if pool["type"] == "data"), None
-    )
+def get_dd_for_each_iteration(client, pool_name):
```
Comment (hkadam134) [inline tests/cephfs/cephfs_scale/max_smallfile_snap.py:14]: what does dd mean ?

Comment (AmarnatReddy) [reply tests/cephfs/cephfs_scale/max_smallfile_snap.py:14]: for each iteration, i wanted to get how many mb data needs to be filled. so i named it as dd.
Changing it to get_data_to_fill_for_each_iteration

Comment (hkadam134) [reply tests/cephfs/cephfs_scale/max_smallfile_snap.py]: Okay

**Review by hkadam134 — DISMISSED** [review]
lgtm

File: tests/cephfs/cephfs_scale/max_smallfile_snap.py [hunk 2]
```diff
@@ -6,18 +6,16 @@
...
 
-def get_available_space(client, fs_name="cephfs"):
-    out, rc = client.exec_command(
-        sudo=True, cmd=f"ceph fs status {fs_name} --format json"
-    )
-    output = json.loads(out)
-    return next(
-        (pool["avail"] for pool in output["pools"] if pool["type"] == "data"), None
-    )
+def get_data_to_fill_for_each_iteration(client, pool_name):
```
Comment (neha-gangadhar) [inline tests/cephfs/cephfs_scale/max_smallfile_snap.py:14]: Please add docstrings to functions.

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**